plugin/trino-hive/src/main/java/io/trino/plugin/hive/HiveConfig.java

```diff
@@ -777,6 +778,19 @@ public HiveConfig setSortedWritingEnabled(boolean sortedWritingEnabled)
         return this;
     }
 
+    public boolean isOptimizeSortedFileReads()
+    {
+        return optimizeSortedFileReads;
+    }
+
+    @Config("hive.optimize-sorted-file-reads")
```

Comment:
Maybe something like `hive.propagate-sorting-properties`?
It's more explicit. Sorted file reads are always optimized since we can do better filtering regardless whether this property is on or not.

Reply:
Changed now to `hive.use-sorting-properties`